src/ssl.c

```diff
@@ -30758,65 +30804,87 @@ int wolfSSL_RSA_sign_ex(int type, const unsigned char* m,
     return ret;
 }
 
-
 /* returns WOLFSSL_SUCCESS on successful verify and WOLFSSL_FAILURE on fail */
 int wolfSSL_RSA_verify(int type, const unsigned char* m,
                                unsigned int mLen, const unsigned char* sig,
                                unsigned int sigLen, WOLFSSL_RSA* rsa)
 {
+    return wolfSSL_RSA_verify_ex(type, m, mLen, sig, sigLen, rsa, RSA_PKCS1_PADDING);
+}
+
+#define wolfSSL_RSA_verify_ex_return(msg, ret_code) {               \
```

Comment:
Macros for cleanup seem problematic. I'd prefer a goto end over this.

Reply:
Todd asked I keep goto's to a minimum but in this case I guess it makes more sense.

Reply:
There are ways to refactor the logic to not use a macro or goto. That flow is usually:

```c
/* proper init of all stack varibles / args */

ret = funa();
if (ret == 0)
   ret = funcb();
if (ret == 0)
   ret = funcc();

/* all caleanup here */
```

Reply:
I used a goto because it is simpler with functions that have different success conditions.